lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -329,11 +332,44 @@ legacyUtxoWitness xpub bytes = TxWitness $ BL.toStrict $ runPut $ do
     putByteString (unXPub xpub)
     putByteString bytes
 
+-- | Decode the contents of a delegated transaction @Transaction@-message.
+getDelegatedCertificateTransaction :: Int -> Get (Tx, [TxWitness])
+getDelegatedCertificateTransaction =
+    getGenericTransaction MsgTypeCertificate "getDelegatedCertificateTransaction"
```

Comment:
Not directly relevant to this PR, but the way we pass the `MsgTypeCertificate` tag here feels wrong to me.

It seems to be for calculating the hash of the message, where the `msgTag` needs to be included.

I think we could do that calculation lazily in `getMessage` and pass the resulting txid in the `2 -> getTransaction` and `3 -> getDelegatedCertificateTransaction` branches.

Reply:
#937

Reply:
ok, complied and readjusted the code to this change :+1: